### PR TITLE
:memo: Add brand guidelines for the brand assets

### PR DESCRIPTION
**Pull request by pragdua**
🚧 

Brand guidelines for the Devfolio brand-assets README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,76 @@
+
 # brand-assets
```
Comment (prateek3255) [inline README.md]: We can remove this

File: README.md [hunk 2]
```diff
@@ -1,4 +1,76 @@
...
-Devfolio Brand Assets
 
-If you're using Devfolio's D as a profile picture and the D doesn't seem to be centered, try using `Devfolio Square/Logo-Square centred.png`
\ No newline at end of file
+  
+
+## Devfolio Brand Assets
```
Comment (prateek3255) [inline README.md]: Change this to a h1(#)

```suggestion
# Devfolio Brand Assets
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,76 @@
...
+
+The following Assets are included;
+
+  
+
+1. `Logo + Text` assets involve the Devfolio 'D' logo with the 'Devfolio' text without any background.
+
+It has three variants -
+
+- `White`<img src ="Logo + Text - White/Devfolio - White.png" style="width: 100px; background-color: #2F3437; padding: 8px 10px 8px 10px; border-radius: 8px;margin-left: 20px; ">
```
Comment (prateek3255) [inline README.md]: This looks kind of weird right now, would look nice if it was center aligned, and had some gap between the text and the logo
image

**Review by prateek3255 — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,4 +1,76 @@
...
+  
+  
+
+#### Do's
+
+1. You may use the brand assets provided in good faith to represent the products and services that we offer.
+
+2. Any uses in hackathons, or any other events organised need to be affiliated with and registered on the devfolio platform.
+
+- Associated advertisements, media, websites with the aforementioned hackathons are permitted to use the Devfolio branding under this condition too.
```
Comment (prateek3255) [inline README.md]: Shouldn't this be:
```suggestion
3 Associated advertisements, media, websites with the aforementioned hackathons are permitted to use the Devfolio branding under this condition too.
```

**Review by prateek3255 — CHANGES_REQUESTED** [review]

**Comment by parzuko** [comment]
@pragdua Would be super helpful if you could share the output markdown content in the PR description!

**Review by prateek3255 — APPROVED** [review]

**Review by chtushar — APPROVED** [review]

**Pull request merged**